src/routes/jira/connect/enterprise/app/jira-connect-enterprise-app-create-or-edit.ts

```diff
@@ -1,16 +1,16 @@
 import { NextFunction, Request, Response } from "express";
 import { GitHubServerApp } from "models/github-server-app";
 
-export const JiraEditAppGet = async (
+export const JiraConnectEnterpriseAppCreateOrEdit = async (
 	req: Request,
 	res: Response,
 	next: NextFunction
 ): Promise<void> => {
 	try {
 		req.log.debug("Received Jira edit app page request");
 
-		const gitHubServerAppId = req.params.id as string;
-		const app = await GitHubServerApp.getForGitHubServerAppId(parseInt(gitHubServerAppId));
+		const uuid = req.params.uuid as string;
+		const app = uuid ? await GitHubServerApp.findForUuid(uuid) : null;
```

Comment:
`const app = await GitHubServerApp.findForUuid(uuid)` should do.  `findForUuid` should look to see if `uuid` is defined, if not should return null

Reply:
yeah, `findForUuid` doesn't do that check.  Maybe add it? just a simple `if(!uuid) { return null; }` will do